examples/get-file-contents.js

```diff
@@ -0,0 +1,112 @@
+import {
+    PrivateKey,
+    FileContentsQuery,
+    Hbar,
+    FileCreateTransaction,
+    FileDeleteTransaction,
+    Client,
+    AccountId,
+} from "@hashgraph/sdk";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * How to get file contents
+ *
+ */
+
+async function main() {
+    /**
+     * Step 0:
+     * Create and configure the SDK Client
+     */
+    if (
+        process.env.OPERATOR_ID == null ||
+        process.env.OPERATOR_KEY == null ||
+        process.env.HEDERA_NETWORK == null
+    ) {
+        throw new Error(
+            "Environment variables OPERATOR_ID, HEDERA_NETWORK, and OPERATOR_KEY are required.",
+        );
+    }
+
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringED25519(process.env.OPERATOR_KEY);
+
+    // Create a client for the local Hedera network
+    const client = Client.forNetwork({
+        [process.env.HEDERA_NETWORK]: new AccountId(3),
+    });
+    client.setOperator(operatorId, operatorKey);
+
+    // Increase the timeout settings
+    client.setRequestTimeout(120000);
```

Comment:
Could you provide more details on why this is necessary? Will it still work without it?

Reply:
this was unnecessary i omitted it was for testing purposes